src/collections/_documentation/platforms/javascript/ember.md

```diff
@@ -3,7 +3,8 @@ title: Ember
 sidebar_order: 50
 ---
 <!-- WIZARD -->
-To use Sentry with your Ember application, you will need to use `@sentry/browser` (Sentry’s browser JavaScript SDK).  
+To use Sentry with your Ember application, you will need to use Sentry’s browser JavaScript SDK: `@sentry/browser`.
```

Comment:
Might be pointless, but swapped the wording so this visually looks better in the wizard.